util/include/util/legendbuilder.h

```diff
@@ -19,7 +19,8 @@ class LegendBuilder
 
   void addNode(
     const std::string& label_,
-    const std::vector<std::pair<std::string, std::string>>& attrs_);
+    const std::vector<std::pair<std::string, std::string>>& attrs_,
+    bool html_ = false);
 
   void addEdge(
```

Comment:
Extend this function with `html_` parameter to be consistent. Extend `setStyle()` functions too.